backend/src/main/java/de/laura/project/service/TriviaService.java

```diff
@@ -0,0 +1,41 @@
+package de.laura.project.service;
+
+import de.laura.project.api.model.TriviaApiData;
+import de.laura.project.api.service.TriviaApiService;
+import de.laura.project.model.TriviaDataPoint;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+
+public class TriviaService {
+
+    private final TriviaApiService triviaApiService;
+
+    @Autowired
+    public TriviaService(TriviaApiService triviaApiService) {
+        this.triviaApiService = triviaApiService;
+    }
+
+    public List<TriviaDataPoint> getListDataPoints() {
+
+        TriviaApiData triviaApiDataArray[] = triviaApiService.getDataPoints();
```

Comment:
you may want to consider using a `List` instead of `[]` - if your API is returning an array you can convert it to an ArrayList like this: 
`new ArrayList<>(Arrays.asList(array));`